### PR TITLE
Cdsid 1225 section 5 23

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Party.scala [hunk 1]
```diff
@@ -19,7 +19,7 @@ package uk.gov.hmrc.cdsimportsddsfrontend.domain
 case class Party(
                   name: Option[String] = Some("French Foil Ltd."),
                   identifier: Option[String] = Some("GB225456994000"),
-                  address: Option[Address] = Some(Address()),
+                  address: Option[Address] = None,
```
Comment (kjainsc) [inline app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Party.scala:22]: default value needed for address? I think, we were planning to remove default values from domain models.

File: app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/LoadingLocationXmlWriter.scala [hunk 2]
```diff
@@ -25,7 +25,7 @@ object LoadingLocationXmlWriter {
   implicit val loadingLocationXmlWriter: XmlWriter[LoadingLocation] = new XmlWriter[LoadingLocation] {
     override def toXmlOption(value: LoadingLocation): Option[Elem] = {
       val id: Node = element("ID", value.id)
-      Some(<LoadingLocation>{id}</LoadingLocation>)
+      Option(<LoadingLocation>{id}</LoadingLocation>)
```
Comment (kjainsc) [inline app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/LoadingLocationXmlWriter.scala:28]: why this is changed, is it necessary as null check not required here?

Comment (retroredge) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/LoadingLocationXmlWriter.scala:28]: We wanted to keep the writers consistent. Most use the `Option` factory to create the optional return type.

Comment (kjainsc) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/services/xml/LoadingLocationXmlWriter.scala:28]: ok, just don't see the point of having extra null check from `Option.apply()` here

File: app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Party.scala (hunk 1 above)
Comment (ellamdav) [reply app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Party.scala:22]: we've moved some defaults from the domain models into the viewmodels, but haven't completely cleaned up Party due to the size of the task

**Pull request merged**